server/app.js

```diff
@@ -274,7 +272,7 @@ app.get('/edit/:id', asyncHandler(getBrew('edit')), (req, res, next)=>{
 	req.ogMeta = { ...defaultMetaTags,
 		title       : req.brew.title || 'Untitled Brew',
 		description : req.brew.description || 'No description.',
-		image       : req.brew.thumbnail || null,
+		image       : req.brew.thumbnail || `${config.get('publicUrl')}/thumbnail.png`,

```

Comment:
what happens if you do:

```suggestion
		image       : req.brew.thumbnail || defaultMetaTags.image,

```